src/app.ts

```diff
@@ -0,0 +1,14 @@
+import Express from 'express'
+
+class App {
```

Comment:
@kbhutani0001 @sharmaaditya570191 how do we feel Object-oriented vs Functional? 
I want you to decide because my personal taste it's too strong to comment on this :P

Reply:
Functional please if we can :p

Reply:
Sure we can. I'm ok with both ways, and everything is achievable using both paradigms.

Reply:
@en3sis @kbhutani0001 so we will go with functional paradigm, right?

Reply:
> @en3sis @kbhutani0001 so we will go with functional paradigm, right?

Yes please =)